### DISCUSSION TITLE
Tip for Flatpak users that have their music on a Network drive (NAS/OMV)

### DISCUSSION THREAD

**Discussion by kevinfiol** [fooyin/fooyin#423] [Show and tell]
First of all, I want to thank @ludouzi for making fooyin. For years I've looked for a serviceable music player on Linux, never quite finding something as comparable to foobar2000. The first day I tried fooyin, I was blown away by the polish, customizability, and huge breadth of features.

## Problem

Anyway, I store all of my music on a Raspberry Pi running OMV attached to my router (so I can access my music on every device in my house).

In `/etc/fstab` I simply run this on startup, mapping the directory to `~/pidrive`:
```
//<LOCAL_IP_ADDRESS_OF_RASPBERRY_PI>/pidrive /home/kevin/pidrive cifs user=kevin,pass=hunter2,x-systemd.automount 0 0
```

If you use Flatpak, `fooyin` will be sandboxed and map drives using the literal `/run/**` directory mapping. This is OK for non-NAS folders.
![2024-12-06_09-42](https://github.com/user-attachments/assets/b99754e2-ee13-4b36-a8e9-9c4321220f4a)

The issue is that between reboots, the `/run/**` path for `/home/kevin/pidrive` will change, and fooyin will fail to find your library:
![2024-12-09_08-55](https://github.com/user-attachments/assets/fa65b928-bf0b-4af4-8a11-16f7bf6814a8)

## Solution
This AskUbuntu answer was helpful: https://askubuntu.com/a/1094903

In short, make sure that you give fooyin access to the directory where your music is.
```
sudo flatpak override org.fooyin.fooyin --filesystem="/home/kevin/pidrive/public/Music"
```

Then you can mount your network directory normally!
![image psd](https://github.com/user-attachments/assets/9abae041-d24f-4a3e-953f-d21cdfcb4c02)

**Comment by bassstorm** [comment]
Thanks for the tip! You're not alone with this kind of setup ;)
